pkg/collector/py/loader.go

```diff
@@ -132,6 +132,9 @@ func (cl *PythonCheckLoader) Load(config integration.Config) ([]check.Check, err
 				log.Errorf("'%s' python wheel attribute '__version__' has the wrong type (%s) instead of 'string'", config.Name, typeName)
 			}
 		} else {
+			// GetAttrString will set an error in the interpretor
+			// if __version__ doesn't exists. We purge it here.
+			glock.getPythonError()
```

Comment:
I think it's worth explicitly handling the return values here, i.e.:
1. logging the python exception string in the `INFO` log message below
2. checking the 2nd return value (which is an error) and log an error if it's not `nil`